lxd/storage/backend_lxd.go

```diff
@@ -1524,13 +1572,20 @@ func (b *lxdBackend) MountInstanceSnapshot(inst instance.Instance, op *operation
 		return false, err
 	}
 
+	// Get the root disk device config.
+	_, rootDiskConf, err := shared.GetRootDiskDevice(inst.ExpandedDevices().CloneNative())
+	if err != nil {
+		return false, err
+	}
+
 	// Get the parent and snapshot name.
 	parentName, snapName, _ := shared.InstanceGetParentAndSnapshotName(inst.Name())
 
-	// Get the volume name on storage.
-	volStorageName := project.Prefix(inst.Project(), parentName)
+	// Get the volume.
+	vol := b.newVolume(volType, InstanceContentType(inst),
```

Comment:
@monstermunchkin I noticed that in some places you've left the `volStorageName` variable defined and passed that to `b.newVolume()` and in other cases, such as here, you've removed it and used the `project.Prefix()` function directly in the args to `b.newVolume()`. If you left the volStorageName vars defined as they are then it would keep the line length down and avoid the overflow.